apiserver/params/storage.go

```diff
@@ -730,3 +730,19 @@ type StorageAddParams struct {
 type StoragesAddParams struct {
 	Storages []StorageAddParams `json:"storages"`
 }
+
+// StorageAttachParams holds the parameters for a Storage.Attach API call.
+type StorageAttachParams struct {
+	Attachments []StorageAttachParam `json:"attachments,omitempty"`
+}
+
+// StorageAttachParam holds a parameter for a Storage.Attach API call.
+type StorageAttachParam struct {
```

Comment:
This is the same as for detaching storage. Can we use a generic struct holding a storage tag and entity tag for both API calls to avoid an explosion in params structs. This has been the required methodology so far for other API params. eg Entities, StringResults, etc